elftools/elf/sections.py

```diff
@@ -500,8 +467,133 @@ def _make_subsections(self):
         self.stream.seek(self.subsec_start)
 
         while self.stream.tell() != end:
-            subsec = ARMAttributesSubsection(self.stream,
-                                             self.structs,
-                                             self.stream.tell())
+            subsec = self.subsection(self.stream,
+                                     self.structs,
+                                     self.stream.tell())
             self.stream.seek(self.subsec_start + subsec['length'])
             yield subsec
+
+
+class ARMAttribute(Attribute):
+    """ ARM attribute object - representing a build attribute of ARM ELF files.
+    """
+    def __init__(self, structs, stream):
+        super(ARMAttribute, self).__init__(
+            struct_parse(structs.Elf_Arm_Attribute_Tag, stream))
+
+        if self.tag in ('TAG_FILE', 'TAG_SECTION', 'TAG_SYMBOL'):
+            self.value = struct_parse(structs.Elf_word('value'), stream)
+
+            if self.tag != 'TAG_FILE':
+                self.extra = []
+                s_number = struct_parse(structs.Elf_uleb128('s_number'), stream)
+
+                while s_number != 0:
+                    self.extra.append(s_number)
+                    s_number = struct_parse(structs.Elf_uleb128('s_number'),
+                                            stream
+                               )
```

Comment:
nit: formatting -- move this `)` to previous line

Reply:
Done.